### PR TITLE
Feature/11897 header

### PR DESCRIPTION
**Pull request by erochOnGit**
Hey ! I already think that i'll be shaken on the navigation file so please tell me how you would have done it ?

### CODE REVIEW

File: src/web/theme/modules/Header/Navigation/Navigation.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    opacity: 0;
+  }
+
+  to {
+    display: unset;
+    opacity: 1;
+  }
+}
+.modal__navigation {
+}
```
Comment (adrien-louis-r) [inline src/web/theme/modules/Header/Navigation/Navigation.scss]: You can remove this empty rule.

File: src/web/theme/ui/atoms/Typography/Link/Link.scss [hunk 2]
```diff
@@ -5,3 +5,8 @@ a {
 a.reversed {
   color: $reverseFontColor;
 }
+a.simple {
+  color: $fontColor;
+  text-decoration: none;
+  font-weight: $regular;
+}
```
Comment (adrien-louis-r) [inline src/web/theme/ui/atoms/Typography/Link/Link.scss]: Nit: you could use a class instead of plain `a` selector.

**Review by aloross — APPROVED** [review]
Only some minors issues, great job :+1:

File: src/web/theme/modules/Header/Navigation/Navigation.js [hunk 3]
```diff
@@ -1,20 +1,55 @@
...
+              <CartModal>
+                {openCart => (
+                  <Button type="invisible" onClick={openCart}>
+                    Cart
+                  </Button>
+                )}
+              </CartModal>
+            </div>
+            <div className="navigation__linklist__label">
+              <Link to={"/orderreference"} type="simple">
```
Comment (hihuz) [inline src/web/theme/modules/Header/Navigation/Navigation.js]: No need for the brackets when the prop is a string :slightly_smiling_face:

File: src/web/theme/modules/Header/Navigation/Navigation.js [hunk 4]
```diff
@@ -1,20 +1,55 @@
...
+      </ModalContext.Provider>
     </nav>
   );
 };
 
-export default Navigation;
+export default compose(
+  withState("isNavigationOpened", "setOpenStateNavigation", false),
+  withHandlers({
+    setOpenStateNavigation: props => () =>
```
Comment (hihuz) [inline src/web/theme/modules/Header/Navigation/Navigation.js]: The name of this sounds a bit convoluted to me, what about "toggleNavigation" ?

File: src/web/theme/modules/Header/Navigation/Navigation.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  &__linklist {
+    box-shadow: 1px 1px 10px $shade03;
+    background: white;
+    animation-duration: 0.5s;
+    animation-name: fadeIn;
+    opacity: 1;
+    position: absolute;
+    right: 1em;
+    top: 1em;
+    display: unset;
```
Comment (hihuz) [inline src/web/theme/modules/Header/Navigation/Navigation.scss]: Why do you need this here ? it also looks duplicated in the animation above. Am I missing something ?

**Review by hihuz — APPROVED** [review]
nice :+1:

**Pull request merged**